docs/questionSharing.md

```diff
@@ -35,6 +35,14 @@ To refer to a question from another course, use the question id (qid) prefixed b
 ]
 ```
 
+## Client and Server Files
+
+If a sharing course attempts to share a question which accesses client or server files associated with a course instance or an assessment, the question will not work as expected because the consuming course can not use it within the context of the sharing course's course intance or assessment.
```

Comment:
Maybe explicitly state what server files can be accessed as well (instead of just those that do not work)? This line is a little vague.